tools/codegen/base/dprint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "repository": "https://github.com/dprint/dprint",
+  "tag_prefix": "",
+  "asset_name": "${package}-${rust_target}.zip",
+  "version_range": ">= 0.34.1",
```

Comment:
Hmm. As for tools after #27, we would like to support basically the latest release at the time of addition and later versions to reduce the manifest size bloat. However, this appears to cause problems similar to https://github.com/taiki-e/install-action/issues/25#issuecomment-1265219010...

Reply:
Added manifest for older versions for now. As for the manifest size issue, I will track it in https://github.com/taiki-e/install-action/issues/36.